client/extensions/woocommerce/state/action-list/test/selectors.js

```diff
@@ -41,13 +43,17 @@ describe( 'selectors', () => {
 			const threeSecondsAgo = Date.now() - 3000;
 			const twoSecondsAgo = Date.now() - 2000;
 
-			const actionList = [
-				{ description: 'Action Step 1', action: { type: 'ACTION1' },
-					startTime: fiveSecondsAgo, endTime: threeSecondsAgo },
-				{ description: 'Action Step 2', action: { type: 'ACTION2' },
-					startTime: twoSecondsAgo },
-				{ description: 'Action Step 3', action: { type: 'ACTION3' } },
-			];
+			const actionList = {
+				steps: [
+					{ description: 'Action Step 1', action: { type: 'ACTION1' },
```

Comment:
Yeah, I was trying to avoid more and more lines as these tests get pretty long already. I guess that's the nature of test data though.